source/extensions/filters/http/cache/cache_headers_utils.h

```diff
@@ -91,6 +92,10 @@ class CacheHeadersUtils {
   // header_entry is null or malformed.
   static SystemTime httpTime(const Http::HeaderEntry* header_entry);
 
+  // Calculates the age of a cached response
+  static std::chrono::seconds calculateAge(const Http::ResponseHeaderMap& response_headers,
+                                           const SystemTime response_time, const SystemTime now);
```

Comment:
I define `using Seconds = std::chrono::seconds` in multiple test files to make them easier to write/read. How about defining it here? Then I can also use it as the return type and inside the function too (also avoids redefining it in multiple test files). Thoughts?